TABTestKit/Classes/BDD/Step.swift

```diff
@@ -21,83 +16,40 @@ public typealias And = Step
 ///
 /// Do not use this Step type directly, instead use one of the typealiases above, like Given, When, Then or And.
 /// To learn more about how to use a Step read this article: https://medium.com/kinandcartacreated/swifty-gherkins-part-1-28abba7dfd8
-public struct Step {
+open class Step {
```

Comment:
How come this needs to be a class now? And does it really need to be subclassable (open)?

Reply:
a struct can not inherit from another struct.
And to reduce code duplication I'm using `String(describing: Swift.type(of: self))` to determine if it's a `Given`, `Then`, `When` or `And` Step

Reply:
Hmm okay, if it's only being subclassed internally then it could be `public` at least?

Reply:
outdated conversation, due to other change